packages/cli/src/commands/widgetset/deploy/index.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { WidgetSetConfig } from "@osdk/foundry-config-json";
+import { MANIFEST_FILE_LOCATION } from "@osdk/widget-api.unstable";
+import type { CommandModule } from "yargs";
+import type { StemmaRepositoryRid } from "../../../net/StemmaRepositoryRid.js";
+import configLoader from "../../../util/configLoader.js";
+import type { CommonWidgetSetArgs } from "../CommonWidgetSetArgs.js";
+import { logWidgetSetDeployCommandConfigFileOverride } from "./logWidgetSetDeployCommandConfigFileOverride.js";
+import type { WidgetSetDeployArgs } from "./WidgetSetDeployArgs.js";
+
+const command: CommandModule<
+  CommonWidgetSetArgs,
+  WidgetSetDeployArgs
+> = {
+  command: "deploy",
```

Comment:
There are some key differences from site deploy here:
- Version cannot be specified as it's tied to the version built into the manifest
- Project will have to be rebuilt (assuming using vite plugin) after updating git tags or package.json depending on auto version strategy
- The upload target for the sites files is either the widget set itself or optionally a code repository that has been authorized to publish the widget set